Sources/App/Core/SearchFilter.swift

```diff
@@ -144,6 +162,12 @@ enum SearchFilterError: Error {
 struct StarsSearchFilter: SearchFilter {
     static var key: String = "stars"
     
+    private static var numberFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .decimal
+        return formatter
+    }()
+    
```

Comment:
Just doing a final review and noticed this. We have a `starsFormatter` available from everywhere in `NumberFormatter+stars.swift`.

Reply:
Thanks for letting me know, I didn't realise this existed (but suppose makes sense). Have pushed up a commit which uses this now.